package.json

```diff
@@ -67,7 +67,17 @@
         }
       ]
     },
+    "fileAssociations": {
```

Comment:
I managed to make fileAssociations a root level key, but I couldn't rewrite the icon for mac unless I added the ext key also.